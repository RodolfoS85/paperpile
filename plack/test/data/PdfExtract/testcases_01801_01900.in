---
file: ../../../PDFREPOSITORY/001801.pdf
---
file: ../../../PDFREPOSITORY/001802.pdf
---
file: ../../../PDFREPOSITORY/001803.pdf
---
file: ../../../PDFREPOSITORY/001804.pdf
---
file: ../../../PDFREPOSITORY/001805.pdf
---
file: ../../../PDFREPOSITORY/001806.pdf
---
file: ../../../PDFREPOSITORY/001807.pdf
---
file: ../../../PDFREPOSITORY/001808.pdf
---
file: ../../../PDFREPOSITORY/001809.pdf
---
file: ../../../PDFREPOSITORY/001810.pdf
---
file: ../../../PDFREPOSITORY/001811.pdf
---
file: ../../../PDFREPOSITORY/001812.pdf
---
file: ../../../PDFREPOSITORY/001813.pdf
---
file: ../../../PDFREPOSITORY/001814.pdf
---
file: ../../../PDFREPOSITORY/001815.pdf
---
file: ../../../PDFREPOSITORY/001816.pdf
---
file: ../../../PDFREPOSITORY/001817.pdf
---
file: ../../../PDFREPOSITORY/001818.pdf
---
file: ../../../PDFREPOSITORY/001819.pdf
---
file: ../../../PDFREPOSITORY/001820.pdf
---
file: ../../../PDFREPOSITORY/001821.pdf
---
file: ../../../PDFREPOSITORY/001822.pdf
---
file: ../../../PDFREPOSITORY/001823.pdf
---
file: ../../../PDFREPOSITORY/001824.pdf
---
file: ../../../PDFREPOSITORY/001825.pdf
---
file: ../../../PDFREPOSITORY/001826.pdf
---
file: ../../../PDFREPOSITORY/001827.pdf
---
file: ../../../PDFREPOSITORY/001828.pdf
---
file: ../../../PDFREPOSITORY/001829.pdf
---
file: ../../../PDFREPOSITORY/001830.pdf
---
file: ../../../PDFREPOSITORY/001831.pdf
---
file: ../../../PDFREPOSITORY/001832.pdf
---
file: ../../../PDFREPOSITORY/001833.pdf
---
file: ../../../PDFREPOSITORY/001834.pdf
---
file: ../../../PDFREPOSITORY/001835.pdf
---
file: ../../../PDFREPOSITORY/001836.pdf
---
file: ../../../PDFREPOSITORY/001837.pdf
---
file: ../../../PDFREPOSITORY/001838.pdf
---
file: ../../../PDFREPOSITORY/001839.pdf
---
file: ../../../PDFREPOSITORY/001840.pdf
---
file: ../../../PDFREPOSITORY/001841.pdf
---
file: ../../../PDFREPOSITORY/001842.pdf
---
file: ../../../PDFREPOSITORY/001843.pdf
---
file: ../../../PDFREPOSITORY/001844.pdf
---
file: ../../../PDFREPOSITORY/001845.pdf
---
file: ../../../PDFREPOSITORY/001846.pdf
---
file: ../../../PDFREPOSITORY/001847.pdf
---
file: ../../../PDFREPOSITORY/001848.pdf
---
file: ../../../PDFREPOSITORY/001849.pdf
---
file: ../../../PDFREPOSITORY/001850.pdf
---
file: ../../../PDFREPOSITORY/001851.pdf
---
file: ../../../PDFREPOSITORY/001852.pdf
---
file: ../../../PDFREPOSITORY/001853.pdf
---
file: ../../../PDFREPOSITORY/001854.pdf
---
file: ../../../PDFREPOSITORY/001855.pdf
---
file: ../../../PDFREPOSITORY/001856.pdf
---
file: ../../../PDFREPOSITORY/001857.pdf
---
file: ../../../PDFREPOSITORY/001858.pdf
---
file: ../../../PDFREPOSITORY/001859.pdf
---
file: ../../../PDFREPOSITORY/001860.pdf
---
file: ../../../PDFREPOSITORY/001861.pdf
---
file: ../../../PDFREPOSITORY/001862.pdf
---
file: ../../../PDFREPOSITORY/001863.pdf
---
file: ../../../PDFREPOSITORY/001864.pdf
---
file: ../../../PDFREPOSITORY/001865.pdf
---
file: ../../../PDFREPOSITORY/001866.pdf
---
file: ../../../PDFREPOSITORY/001867.pdf
---
file: ../../../PDFREPOSITORY/001868.pdf
---
file: ../../../PDFREPOSITORY/001869.pdf
---
file: ../../../PDFREPOSITORY/001870.pdf
---
file: ../../../PDFREPOSITORY/001871.pdf
---
file: ../../../PDFREPOSITORY/001872.pdf
---
file: ../../../PDFREPOSITORY/001873.pdf
---
file: ../../../PDFREPOSITORY/001874.pdf
---
file: ../../../PDFREPOSITORY/001875.pdf
---
file: ../../../PDFREPOSITORY/001876.pdf
---
file: ../../../PDFREPOSITORY/001877.pdf
---
file: ../../../PDFREPOSITORY/001878.pdf
---
file: ../../../PDFREPOSITORY/001879.pdf
---
file: ../../../PDFREPOSITORY/001880.pdf
---
file: ../../../PDFREPOSITORY/001881.pdf
---
file: ../../../PDFREPOSITORY/001882.pdf
---
file: ../../../PDFREPOSITORY/001883.pdf
---
file: ../../../PDFREPOSITORY/001884.pdf
---
file: ../../../PDFREPOSITORY/001885.pdf
---
file: ../../../PDFREPOSITORY/001886.pdf
---
file: ../../../PDFREPOSITORY/001887.pdf
---
file: ../../../PDFREPOSITORY/001888.pdf
---
file: ../../../PDFREPOSITORY/001889.pdf
---
file: ../../../PDFREPOSITORY/001890.pdf
---
file: ../../../PDFREPOSITORY/001891.pdf
---
file: ../../../PDFREPOSITORY/001892.pdf
---
file: ../../../PDFREPOSITORY/001893.pdf
---
file: ../../../PDFREPOSITORY/001894.pdf
---
file: ../../../PDFREPOSITORY/001895.pdf
---
file: ../../../PDFREPOSITORY/001896.pdf
---
file: ../../../PDFREPOSITORY/001897.pdf
---
file: ../../../PDFREPOSITORY/001898.pdf
---
file: ../../../PDFREPOSITORY/001899.pdf
---
file: ../../../PDFREPOSITORY/001900.pdf
