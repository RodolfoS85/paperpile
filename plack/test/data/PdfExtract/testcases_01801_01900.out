---
authors: 'Starr, D. Barry and Hoopes, Barbara C. and Hawley, Diane K.'
title: DNA Bending is an Important Component of Site-specific Recognition by the TATA Binding Protein
---
authors: 'Steffen, Nicholas R. and Murphy, Scott D. and Lathrop, Richard H.'
title: The Role of DNA Deformation Energy at Individual Base Steps for the Identification of DNA-Protein Binding Sites
---
authors: 'Sumedha,  and Martin, Olivier C and Peliti, Luca'
doi: 10.1088/1742-5468/2007/05/P05011
title: Population size effects in evolutionary dynamics on neutral networks and toy landscapes
---
authors: 'Suppmann, Sabine and Persson, Britt C. and Bock, August'
title: Dynamics and efficiency in vivo of UGA-directed selenocysteine insertion at the ribosome
---
authors: 'Tahirov, Tahir H. and Temiakov, Dmitry and Anikin, Michael and Patlan, Vsevolod and McAllister, William T. and Vassylyev, Dmitry G. and Yokoyama, Shigeyuki'
title: Structure of a T7 RNA polymerase elongation complex at 2.9 A resolution
---
authors: 'Tremousaygue, Dominique and Manevski, Alexandra and Bardet, Claude and Lescure, Nicole and Lescure, Bernard'
title: Plant interstitial telomere motifs participate in the control of gene expression in root meristems
---
authors: 'Ule, Jernej and Jensen, Kirk B. and Ruggiu, Matteo and Mele, Aldo and Ule, Aljaz and Darnell, Robert B.'
title: CLIP Identifies Nova-Regulated RNA Networks in the Brain
---
authors: 'Ule, Jernej and Ule, Aljaz and Spencer, Joanna and Williams, Alan and Hu, Jing-Shan and Cline, Melissa and Wang, Hui and Clark, Tyson and Fraser, Claire and Ruggiu, Matteo and Zeeberg, Barry R and Kane, David and Weinstein, John N and Blume, John and Darnell, Robert B'
doi: 10.1038/ng1610
title: Nova regulates brain-specific splicing to shape the synapse
---
authors: 'Vogan, Kyle J. and ALAN UNDERHILL, D. and Gros, Philippe'
title: An Alternative Splicing Event in the Pax-3 Paired Domain Identifies the Linker Region as a Key Determinant of Paired Domain DNA-Binding Activity
---
authors: 'Wang, Fugao and Landau, D. P.'
doi: 10.1103/PhysRevLett.86.2050
title: 'Efficient, Multiple-Range Random Walk Algorithm to Calculate the Density of States'
---
authors: 'Weckwerth, Wolfram and Selbig, Joachim'
doi: 10.1016/S0006-291X0301182-3
title: Scoring and identifying organism-specific functional patterns and putative phosphorylation sites in protein sequences using mutual information
---
authors: 'Yang, Jiong and Uiuc, Wei Wang and Hill, Unc Chapel'
title: 'CLUSEQ: Efficient and Effective Sequence Clustering'
---
authors: 'Will, Sebastian and Busch, Anke and Backofen, Rolf'
title: Efficient Sequence Alignment with Side-Constraints by Cluster Tree Elimination
---
authors: 'Wolfinger, Michael T and Andreas Svrcek-Seiler, W and Flamm, Christoph and Hofacker, Ivo L and Stadler, Peter F'
doi: 10.1088/0305-4470/37/17/005
title: Efficient computation of RNA folding dynamics
---
authors: 'Xu, Xian and Zhang, Aidong'
title: 'Virtual Gene: a Gene Selection Algorithm for Sample Classification on Microarray Datasets'
---
authors: 'Yang, Hui and Parthasarathy, Srinivasan and Mehta, Sameep'
title: A Generalized Framework for Mining Spatio-temporal Patterns in Scientific Data
---
authors: 'Yang, Hui'
title: Mining Spatial and Spatio-temporal Patterns in Scientific Data
---
authors: 'Yeang, Chen-Hsiang and Ramaswamy, Sridhar and Tamayo, Pablo and Mukherjee, Sayan and Rifkin, Ryan M. and Angelo, Michael and Reich, Michael and Lander, Eric and Mesirov, Jill and Golub, Todd'
title: Molecular classification of multiple tumor types
---
authors: 'a, Yingze Zhang and Gorry a, Michael C. and Christopher Post a, J. and Ehrlich a, Garth D.'
title: Genomic organization of the human fibroblast growth factor receptor 2 (FGFR2) gene and comparative analysis of the human FGFR gene family
---
authors: 'Zhang, Zemin and Wood, William I.'
title: A profile hidden Markov model for signal peptides generated by HMMER
---
authors: 'Zhao, Yong and Samal, Eva and Srivastava, Deepak'
doi: 10.1038/nature03817
title: Serum response factor regulates a muscle-specific microRNA that targets Hand2 during cardiogenesis
---
authors: 'Zhong, Shi'
title: A Unified Framework for Model-based Clustering
---
authors: 'Zhu, Jun and Shendure, Jay and Mitra, Robi D. and Church, George M.'
title: Single Molecule Profiling of Alternative Pre-mRNA Splicing
---
authors: 'Will, Sebastian and Reiche, Kristin and Hofacker, Ivo L. and Stadler, Peter F. and Backofen, Rolf'
doi: 10.1371/journal.pcbi.0030065
title: Inferring Noncoding RNA Families and Classes by Means of Genome-Scale Structure-Based Clustering
---
authors: 'Varani, Luca and Gunderson, Samuel I. and Mattaj, Iain W. and Kay, Lewis E. and Neuhaus, David and Varani, Gabriele'
title: 'The NMR structure of the 38 kDa U1A protein - PIE RNA complex reveals the basis of cooperativity in regulation of polyadenylation by human U1A protein'
---
arxivid: cond-mat/9611065
authors: 'Veitshans, T. and Klimov, D. K. and Thirumalai, D.'
title: 'Protein Folding Kinetics: Time Scales, Pathways, and Energy Landscapes in Terms of Sequence Dependent Properties'
---
authors: 'Xing, Yi and Lee, Christopher'
doi: 10.1038/nrg1896
title: 'Alternative splicing and RNA selection pressure - evolutionary consequences for eukaryotic genomes'
---
authors: 'Barták, Roman'
title: 'Constraint Programming - What is behind?'
---
authors: 'Gerstein, Mark and Jansen, Ronald'
title: 'The current excitement in bioinformatics - analysis of whole- genome expression data: how does it relate to protein structure and function?'
---
authors: 'Jossinet, Fabrice and Westhof, Eric'
title: 'Sequence to Structure (S2S) : display, manipu- late and interconnect RNA data from sequence to structure'
---
authors: 'Kiziltan, Zeynep and Smith, Barbara M.'
title: Symmetry-Breaking Constraints for Matrix Models
---
authors: 'Deutsch, Michael and Long, Manyuan'
title: Intron-exon structures of eukaryotic model organisms
---
authors: 'Falquet, Laurent and Pagni, Marco and Bucher, Philipp and Hulo, Nicolas and Sigrist, Christian J. A. and Hofmann, Kay and Bairoch, Amos'
title: 'The PROSITE database, its status in 2002'
---
authors: 'Kolchanov, N. A. and Ignatieva, E. V. and Ananko, E. A. and Podkolodnaya, O. A. and Stepanenko, I. L. and Merkulova, T. I. and Pozdnyakov, M. A. and Podkolodny, N. L. and Naumochkin, A. N. and Romashchenko, A. G.'
title: 'Transcription Regulatory Regions Database (TRRD): its status in 2002'
---
authors: 'Man, Tsz-Kwong and Stormo, Gary D.'
title: Non-independence of Mnt repressor-operator interaction determined by a new quantitative multiple fluorescence relative affinity (QuMFRA) assay
---
authors: 'Mirny, Leonid A. and Gelfand, Mikhail S.'
title: Structural analysis of conserved base pairs in protein-DNA complexes
---
authors: 'Mobley, Evelyn M. and Pan, Tao'
title: Design and isolation of ribozyme-substrate pairs using RNase P-based ribozymes containing altered substrate binding sites
---
authors: 'Salgado, Heladia and Santos-Zavaleta, Alberto and Gama-Castro, Socorro and Millán-Zárate, Dulce and Blattner, Frederick R. and Collado-Vides, Julio'
title: 'RegulonDB (version 3.0): transcriptional regulation and operon organization in Escherichia coli K-12'
---
authors: 'Shigemoto, Kazuhiro and Brennan, Jane and Walls, Elizabeth and Watson, Christine J. and Stott, David and Rigby, Peter W. J. and Reith, Alastair D.'
title: Identification and characterisation of a developmentally regulated mammalian gene that utilises -1 programmed ribosomal frameshifting
---
authors: 'Gravemeier, V.'
title: The Variational Multiscale Method for Laminar and Turbulent Flow
---
authors: 'Kornberg, Roger D. and Stryer, Lubert'
title: 'Statistical distributions of nucleosomes: nonrandom locations by, stochastic mechanism'
---
authors: 'Mazer, Greg and Booth, Derek and Ewing, Kern'
title: Limitations to vegetation establishment and growth in biofiltration swales
---
authors: 'Sumitomo, Katsuhiko and Li, Tuoping and Hisamatsu, Tamotsu'
doi: 10.1016/j.plantsci.2009.02.003
title: 'Gibberellin promotes flowering of chrysanthemum by upregulating CmFL, a chrysanthemum FLORICAULA/LEAFY homologous gene'
---
authors: 'Draud, M. J. and Verga, J. N. and Haley, M. P. and Itzkowitz, M.'
doi: 10.1007/s10211-007-0036-8
title: Mate inspection patterns in the female beaugregory damselfish (Stegastes leucostictus)
---
authors: 'Kotiaho, Janne S. and Alatalo, Rauno V. and Mappes, Johanna and Parri, Silja'
title: Honesty of agonistic signalling and effects of size and motivation asymmetry in contests
---
authors: 'Mocz, Gabor'
title: A Changing Research and Publication Landscape for Biochemistry
---
authors: 'Tohse, Hidekazu and Mugiya, Yasuo'
doi: 10.3354/ab00029
title: 'Sources of otolith carbonate: experimental determination of carbon incorporation rates from water and metabolic CO, and their diel variations'
---
authors: 'Polte, Patrick and Buschbaum, Christian'
doi: 10.3354/ab00071
title: 'Native pipefish Entelurus aequoreus are promoted by the introduced seaweed Sargassum muticum in the northern Wadden Sea, North Sea'
---
authors: 'Yokes, Baki and Galil, Bella S.'
doi: 10.3391/ai.2006.1.3.5
title: 'Touchdown - first record of Percnon gibbesi (H. Milne Edwards, 1853) (Crustacea: Decapoda: Grapsidae) from the Levantine coast'
---
authors: 'Makarevich, Tamara A. and Mastitsky, Sergey E. and Savich, Irina V.'
doi: 10.3391/ai.2008.3.3.3
title: Phytoperiphyton on the shells of Dreissena polymorpha (Pallas) in Lake Naroch
---
authors: 'Malipatil, MB'
title: 'One new genus and three new species of Orsillinae (Hemiptera: Lygaeidae) from Australia'
---
authors: 'Duffy, Michael P and Nahrung, Helen F and Lawson, Simon A and Clarke, Anthony R'
doi: 10.1111/j.1440-6055.2008.00657.x
title: 'Direct and indirect effects of egg parasitism by Neopolycystus Girault sp. (Hymenoptera: Pteromalidae) on Paropsis atomaria Olivier (Coleoptera: Chrysomelidae)'
---
authors: 'Granadeiro, Jose P. and Bolton, Mark and Silva, Monica C. and Nunes, Manuela and Furness, Robert W.'
title: Responses of breeding Cory's shearwater Calonectris diomedea to experimental manipulation of chick condition
---
authors: 'vulpinus, Buteo buteo'
title: 'Optimal flight behavior of soaring migrants: a case study of migrating steppe buzzards'
---
authors: 'Collier, Grant S. and Pratt, John M. and DE WETt, Christiaan R.'
title: 'Studies, Haemiin in Dimethyl Sulphoxide/Water Mixtures'
---
authors: 'Begona Ruiz-Larrea, M.'
title: A Simple Question to Think about When Considering the Hemoglobin Function
---
authors: 'Guralnick, Robert and Neufeld, David'
title: 'CHALLENGES BUILDING ONLINE GIS SERVICES TO SUPPORT GLOBAL BIODIVERSITY MAPPING AND ANALYSIS: LESSONS FROM THE MOUNTAIN AND PLAINS DATABASE AND INFORMATICS PROJECT'
---
authors: 'Sautter, Guido and BÖHM, Klemens and ONAT AGOSTI, D'
title: A QUANTITATIVE COMPARISON OF XML SCHEMAS FOR TAXONOMIC PUBLICATIONS
---
authors: 'Troyanskaya, Olga G. and Arbell, Ora and Koren, Yair and Landau, Gad M. and Bolshoy, Alexander'
title: 'Sequence complexity profiles of prokaryotic genomic sequences: A fast algorithm for calculating linguistic complexity'
---
authors: 'Diez-Rivero, Carmen M. and García-Boronat, María and Reche, Pedro A'
title: Integrating T-cell epitope annotations with sequence and structural information using DAS
---
authors: 'Poulin, Robert'
title: Evolutionary trends in body size of parasitic flatworms
---
authors: 'Bleiweiss, Robert'
title: 'Tempo and mode, hummingbird evolution'
---
authors: 'Kay, Bartholomew'
title: 'MISCONCEPTIONS: BICARBONATE AS AN ERGOGENIC AID? A PHYSICAL, CHEMICAL, MECHANISTIC VIEWPOINT'
---
authors: 'Vuillaume, Marie-Laure and Uhrhammer, Nancy and Vidal, Véronique and Vidal, Valérie Sylvain and Chabaud, Valérie and Jesson, Beline and Kwiatkowski, Fabrice and Bignon, Yves-Jean'
title: Use of Gene Expression Profiles of Peripheral Blood Lymphocytes to Distinguish BRCA1 Mutation Carriers in High Risk Breast Cancer Families
---
authors: 'Anderson, Carol M. and Blackburn, Elizabeth H.'
title: Mec1 function in the DNA damage response does not require its interaction with Tel2
---
authors: 'Hayashi, Takeshi and Fujita, Yohta and Iwasaki, Osamu and Adachi, Yoh and Takahashi, Kohta and Yanagida, Mitsuhiro'
doi: 10.1016/j.cell.2004.09.002
title: Mis16 and Mis18 Are Required for CENP-A Loading and Histone Deacetylation at Centromeres
---
authors: 'Hemalatha, G. R. and Satyanarayana Rao, D. and Guruprasad, L.'
doi: 10.1155/2007/47161
title: Identification and Analysis of Novel Amino-Acid Sequence Repeats in Bacillus anthracis str. Ames Proteome Using Computational Tools
---
authors: 'Keller, Cherry M. E.'
title: POTENTIAL ROADSIDE BIASES DUE TO HABITAT CHANGES ALONG BREEDING BIRD SURVEY ROUTES'
---
title: Probability of Nonindigenous Seafood Species Becoming Established
---
authors: 'RE¸IT S, H. and AKÇAKAYA,  and Ferson, Scott and Burgman, Mark A. and Keith, David A. and Mace, Georgina M. and Todd, Charles R.'
title: Making Consistent IUCN Classifications under Uncertainty
---
authors: 'Tuno, Nobuko'
title: 'Insect feeding on spores of a bracket fungus, Elfvingia applanata (Pers.) Karst. (Ganodermataceae, Aphyllophorales)'
---
authors: 'Kukalova-Peck, Jarmila'
doi: 10.1007/s11692-007-9013-4
title: 'Phylogeny of Higher Taxa in Insecta: Finding Synapomorphies in the Extant Fauna and Separating Them from Homoplasies'
---
authors: 'Spijkerman, Elly and Barua, Deepak and Gerloff-Elias, Antje and Kern, Jurgen and Gaedke, Ursula and Heckathorn, Scott A.'
doi: 10.1007/s00792-007-0067-0
title: Stress responses and metal tolerance of Chlamydomonas acidophila in metal-enriched lake water and artificial medium
---
authors: 'Munday, Philip L and Jones, Geoffrey P and Pratchett, Morgan S and Williams, Ashley J'
title: Climate change and the future for coral reef fishes
---
authors: 'Pauly, Daniel'
title: 'Charles Darwin, ichthyology and the species concept'
---
authors: 'Hodkinson, D. J. and Askew, A. P. and Thompson, K. and Hodgson, J. G. and Bakker, J. P. and Bekker, R. M.'
title: Ecological correlates of seed size in the British flora
---
authors: 'Goldstein, Bruce Evan'
doi: 10.1007/s10745-007-9133-6
title: 'Skunkworks in the Embers of the Cedar Fire: Enhancing Resilience in the Aftermath of Disaster'
---
authors: 'Salas-Leiva, Dayana Elizabeth and Mayor-Duran, Victor Manuel and Toro-Perea, Nelson'
doi: 10.1007/s10750-008-9611-x
title: 'Genetic diversity of black mangrove (Avicennia germinans) in natural and reforested areas of Salamanca Island Parkway, Colombian Caribbean'
---
authors: 'Martin, Pablo R. and Estebenet, Alejandra L. and Burela, Silvana'
doi: 10.1007/s10750-005-1825-6
title: 'Factors affecting the distribution and abundance of the commensal Temnocephala iheringi (Platyhelminthes: Temnocephalidae) among the southernmost populations of the apple snail Pomacea canaliculata'
---
authors: 'Ferguson, A. J. D.'
title: The role of modelling in the control of toxic blue-green algae
---
authors: 'Sanvicente-Anorve, L. and Alatorre, M. A. and Flores-Coto, C. and Alba, C.'
title: 'Relationships between fish larvae and siphonophores in the water column: effect of wind-induced turbulence and thermocline depth'
---
authors: 'Vamsee-Krishna, C. and Phale, Prashant S.'
title: Bacterial degradation of phthalate isomers and their esters
---
authors: 'Lal, Rup and Dadhwal, Mandeep and Kumari, Kirti and Sharma, Pooja and Singh, Ajaib and Kumari, Hansi and Jit, Simran and Gupta, Sanjay Kumar and Nigam, Aeshna and Lal, Devi and Verma, Mansi and Kaur, Jaspreet and Bala, Kiran and Jindal, Swati'
title: 'Pseudomonas sp. to Sphingobium indicum: a journey of microbial degradation and bioremediation of Hexachlorocyclohexane'
---
authors: 'Bleckmann, Horst and Zelick, Randy'
doi: 10.1111/j.1749-4877.2008.00131.x
title: Lateral line system of fish
---
authors: 'Elvert, Ralf and Ehrhardt, Nicole and Taube, Martin and Klingenspor, Martin and GAILUS-DURNER, Valerie and Fuchs, Helmut and Hrabe DE ANGELIS, Martin and Blanquet, Veronique and Heldmaier, Gerhard'
doi: 10.1111/j.1749-4877.2006.00026.x
title: Metabolic phenotyping of mouse mutants in the German Mouse Clinic
---
authors: 'Li, Yijia'
title: 'Cloning and Characterization of NAD-dependent Deacetylase Sirtuin 2 Homolog from the Silkworm, Bombyx mori'
---
authors: 'Navas, K. A. and Archana Thampy, S. and Sasikumar, M.'
title: EPR Hiding in Medical Images for Telemedicine
---
authors: 'Dhouib, D. and Naït-Ali, A. and Olivier, C. and Naceur, M. S.'
title: Performance Evaluation of Wavelet Based Coders on Brain MRI Volumetric Medical Datasets for Storage and Wireless Transmission
---
authors: 'Mathura, Venkatarajan Subramanian and Paris, Daniel and Ait-ghezala, Ghania and Gupta, Pankaj and Volmar, Claude-Henry and Mullan, Michael'
title: A vector method for comparison of RNA and protein expression levels of cytokines upon CD40L treatment of microglia
---
authors: 'Tripathi, Kaushlendra and Muralikrishna, Bhattiprolu and Parnaik, Veena K'
title: Differential dynamics and stability of lamin A rod domain mutants
---
authors: 'Ballarin, L and Menin, A and Franchi, N and Bertoloni, G and Cima, F'
title: Morula cells and non-self recognition in the compound ascidian Botryllus schlosseri
---
authors: 'Faruki, SI'
title: 'Effect of pyridoxine on the reproduction of the mulberry silkworm, Bombyx mori L'
---
authors: 'Mannello, F and Tonti, G and Papa, S'
title: Are matrix metalloproteinases the missing link?
---
authors: 'Trombetta, Maria Federica and Mattii, Simona and Pasquini, Marina and Falaschini, Adalberto'
title: 'Influence of diet and rearing system on heavy pig performance, carcass and meat quality'
---
authors: 'Zucali, Maddalena and Bava, Luciana and Sandrucci, Anna and Tamburini, Alberto and Piccinini, Renata and Daprà, Valentina and Tonni, Mirella and Zecconi, Alfonso'
title: 'Milk flow pattern, somatic cell count and teat apex score in primiparous dairy cows at the beginning of lactation'
---
authors: 'Yuksekdag, Zehra Nur and Beyatli, Yavuz'
title: Production of Poly-beta-hydroxybutyrate (PHB) in Different Media by Streptococcus thermophilus Ba21S Strain
---
authors: 'Kohler, Florian and Verhulst, Jort and van Klink, Roel and Kleijn, David'
doi: 10.1111/j.1365-2664.2007.01394.x
title: At what spatial scale do high-quality habitats enhance the diversity of forbs and pollinators in intensively farmed landscapes?
---
authors: 'Kalinowski, Andrzej and Klimko, Ma³gorzata and He, Liping and Kluza, Magdalena'
title: 'Two-dimensional electrophoresis patterns of total proteins, esterases and acid phosphatases from pollen, seeds and leaves of three cultivars of Helianthus annuus L'
---
authors: 'Ceccarelli, Daniela and Daccord, Aurelie and Rene, Melissa and Burrus, Vincent'
doi: 10.1128/JB.00150-08
title: Identification of the Origin of Transfer (oriT) and a New Gene Required for Mobilization of the SXT/R391 Family of Integrating Conjugative Elements
---
authors: 'Hall, Brian K'
title: 'The neural crest and neural crest cells: discovery and significance for theories of embryonic organization'
